examples/pxScene2d/src/pxFont.cpp

```diff
@@ -883,6 +883,7 @@ bool pxFontAtlas::addGlyph(uint32_t w, uint32_t h, void* buffer, GlyphTextureEnt
   {
     mTexture = context.createTexture(PXSCENE_FONT_ATLAS_DIM,PXSCENE_FONT_ATLAS_DIM,PXSCENE_FONT_ATLAS_DIM,PXSCENE_FONT_ATLAS_DIM, NULL);
   }
+  rtLogDebug("adding a texture of [%d] size memory", w*h);
```

Comment:
w*h will be unsigned but %d expects signed.

Reply:
Fixed it with %u